### DISCUSSION TITLE
OMG It was there all along! Control Panel Column Filters

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js-ui#1836] [General]
Well, well. I discovered today, quite by accident, that if you click on the little upside down triangle next to a column header in the CP that you get a drop down where you can type in what you want filter on. I had only discovered the sort functionality, and have been _frustrated_ that I could not search or filter by text. Guess I should have RTFM. :-/ 

Am I the only one?

**Comment by robertsLando** [comment]
You can also group by some columns :) Maybe you have any suggestion to make it more clear? You can also filter wich columns to display and which not, and the columns order you prefer
